### ISSUE TITLE
Create Competitor Booklet

### ISSUE THREAD

**Issue by mildlyincompetent** [srobo/tasks#87]
We should design an event booklet which can be provided to competitors and which contains an overview of the event.

This should include:
- a detailed schedule (probably the same as the website has); see #73
- a reminder of any important rules or regulations
- instructions on how robot inspection will work
- (optional) suggestions of where to go for lunch

Blocking on #73 #65.

Note: this is not the flyers for guests, which is covered by #101.

**Comment by PeterJCLaw** [comment]
https://studentrobotics.org/cgit/marketing.git _might_ be useful here, though could well also be far too outdated.
https://studentrobotics.org/style/ almost certainly is useful.

I think that @richardbarlow has designed these in the past and may be able to provide resources.

Other content these typically contain:
- outline of the rules
- list of teams (TLAs, names/schools)

**Comment by richardbarlow** [comment]
The design for the SR2016 leaflet can be found in arena.git (not sure why I put it in there): https://git.studentrobotics.org/cgit/arena.git/tree/comp16

I think that @jimmythompson designed the SR2017 leaflet and got them printed.

The README in the comp16 directory includes information relevant to getting them printed.

**Comment by Tyler-Ward** [comment]
Leadtime: 1 week

**Comment by WillB97** [comment]
Probably also blocked on #123

**Comment by Tyler-Ward** [comment]
Current thoughts for layout

- [ ] Cover: some nice graphics.
- [ ] Inside front cover: message from the core team/trustees
- [ ] P1: The game
- [ ] P2: The Teams
- [ ] P3: Venue map
- [ ] P4: Schedule
- [ ] P5: Internet access/similar notes. (could have place for a username/password sticker here)
- [ ] P6: Food map
- [ ] P7: Social media
- [ ] P8: Get involved/Volunteer recruitment
- [ ] inside rear cover: Sponsors

**Comment by PeterJCLaw** [comment]
Thanks for getting us started here @Tyler-Ward.

> * [ ]  P1: The game
> * [ ]  P2: The Teams

I'm not sure what size you're thinking to print this on, though I'd be quite surprised if we needed a whole page for each of these?

> * [ ]  P3: Venue map

This might need more than a single page, particularly if we want it to be at any useful scale.
A related question: do we want to have copies of this available for visitors? If so, maybe leave this out of the booklet specifically and then just drop in a couple of copies loose?

> * [ ]  P4: Schedule

Do you mean the high level schedule (per the event page on the website) or the matches? I'd rather not include the matches in anything printed as they're likely to change, even after the point where the competitors have arrived at the venue. (This is a _feature_ of the system).

> * [ ]  P5: Internet access/similar notes. (could have place for a username/password sticker here)

Like the idea of a slot for a credentials sticker :+1:
Not sure this needs a whole page though? Maybe have a slot for this sticker on the front cover and drop this page?
That way it can double as the label for which teams it's meant for.

**Comment by Tyler-Ward** [comment]
For the core team message i have created a document here, alterations/alternate introductions welcomed. We should also reach out to the trustees and see if they want a message on this page as well.

https://docs.google.com/document/d/1F8-6JplKfB1_47nMyHCsbJUEhS4YgDI_Tc5DBcEUD_4/edit

**Comment by Tyler-Ward** [comment]
For timings on this we will need to order these on/around the 25th to ensure we get them in time so the designs should be ready for review some-point around the middle of next week.

**Comment by trickeydan** [comment]
> For the core team message i have created a document here, alterations/alternate introductions welcomed.

Alternative written by me.

**Comment by Tyler-Ward** [comment]
I have created a gitrepo to store this which can be found at https://github.com/srobo/competitor-booklet-2019

**Comment by Tyler-Ward** [comment]
In order to get this moving we will need some content filling in.

* @prophile Are you able to take a look at writing a brief game summary will also be used for #101 and will be useful for commentators and SUSU TV
* @RealOrangeOne can you produce a final list of TLAs and school names for including in the book and #101 this can also feed into #123 
* #92 is in progress waiting on data from #123  ready to be merged into this
* #73 is mostly done just waiting on timing details for #120 
* If anyone has any things that want to go on the notes page the  add them to the doc.
* @jcwilliamson can you convert the food information you gathered into a summary for the booklet we can also put this on the event page as part of #160 
* @antoinepetty Can you fill in the details on the social media info
* @Adimote Can you take a look at the getting involved text
* @TomWheal can you take a look at creating the graphical style/page layout of the document (feel free to heavily edit my outline documents.

**Comment by RealOrangeOne** [comment]
> can you produce a final list of TLAs and school names

This list already exists in Google Drive. Simply exclude those marked as 'Dropped out'

**Comment by antoinepetty** [comment]
Not sure how wordy this is expected to be. I've added a short bit on social media info in the Google Doc.

**Comment by trickeydan** [comment]
Fun Idea (time sensitive though): We could ask teams to submit a name and short "bio" / "description" of their robot. A lot of teams (e.g CLY, SWI) have named and almost given a personality of sorts to their robot in the past and I think that this could be a fun, light-hearted thing to but in the booklet so that teams learn about each other's robots. 

I'm unsure about the length that this would take up and is quite dependent on formatting. I've never seen a booklet for a previous idea so am just throwing ideas around. @Tyler-Ward

**Comment by Scarzy** [comment]
Does this require any information from #125 ?

**Comment by Tyler-Ward** [comment]
@Scarzy good spot, It would make sense to include these in the booklet If we put P5 and P7 together as these are looking quite short and will fit well together, that will leave one page free for the rules. Are you able to write these up?

**Comment by Tyler-Ward** [comment]
I have emailed the trustees to get the trustee intro paragraph from them.

**Comment by Tyler-Ward** [comment]
@trickeydan That is an interesting idea while I don't expect we will have room for a bio/description without adding extra pages (could add an extra 4 pages if people think this is a good idea and we can get the data in time) we should have room for a team name or similar on the team pages with the TLA and school name which would make sense to include if we have the data. We had a set of teams info on the website in previous years but I'm not sure if we have this info or have collected team names from teams. @RealOrangeOne @PeterJCLaw might know if this has happened. I believe this data is also fed into #119 but could be collected on the day for this use.

**Comment by PeterJCLaw** [comment]
We used to collect team names/info via the "Team Status" mechanism in the IDE. Historically that then appeared on our website as well as this sort of thing. Since the new website doesn't include this, it's not active in the IDE and we don't have any data.

Given the timings we have here, I suggest punting this aspect to another year and looking to get the Team Status system back up & running (https://github.com/srobo/website/issues/33).

**Comment by PeterJCLaw** [comment]
> * @prophile Are you able to take a look at writing a brief game summary will also be used for #101 and will be useful for commentators and SUSU TV if they do the livestream

Alternatively use the the spiel from https://studentrobotics.org/news/2018-11-14-sr2019-rules-published/.

(I remain unconvinced that this section is remotely useful to _competitors_ though -- they should already know the game _very_ well by now)

**Comment by PeterJCLaw** [comment]
As a meta point, could I encourage that we focus on https://github.com/srobo/tasks/issues/101 which personally I think is a higher priority than this booklet. The teams know why they're at the competition while guests need a lot more of an introduction. Additionally, any prep for that ticket also benefits competitors (but not so much the other way around).

**Comment by TomWheal** [comment]
Progress on this is blocked by the information needed to go into the booklet

**Comment by Tyler-Ward** [comment]
A list of filtered teams (Just TLA,name, and rookie status) is here.

https://docs.google.com/spreadsheets/u/1/d/1KVIMpXJ_9jPMLbg-hSe9WCY-D5Unru49LJVH6q-JD20/edit#gid=0

**Comment by jcwilliamson** [comment]
#79 Should be linked here. This now contains a google my map showing local destinations.

**Comment by TomWheal** [comment]
[Test-Booklet.pdf](https://github.com/srobo/tasks/files/2999608/Test-Booklet.pdf)
A version without some of the sections done.

Something to note, this will be the colour it will be printed. There is no CYMK exact version of the current sr style colour.

**Comment by PeterJCLaw** [comment]
nit: our [house style](https://studentrobotics.org/style/) is that headings be in UPPERCASE (in Oswald or Open Sans Condensed, depending on their level)

**Comment by TomWheal** [comment]
This currently I what ill send to print, without the food map, venue map and any quotes as I'm waiting on those / peoples opinion on them.

[Booklet-v0.pdf](https://github.com/srobo/tasks/files/3014448/Booklet-v0.pdf)

**Comment by Scarzy** [comment]
Page 5 should read "rookie teams"

Scarzy

**Comment by trickeydan** [comment]
The encoding on the Food Map is somewhat broken.

Also "Sponsers" -> "Sponsors" on 2nd last page.

**Comment by RealOrangeOne** [comment]
On the title page, can we change the emphasis on the year and game titles, I suspect 1 should be much larger than the other. Maybe swap the order, shrink the year, and make the game title italic (context: I'm not a designer, not should I ever be).

Excluding the above comments, this looks great!

**Comment by Tyler-Ward** [comment]
A few thoughts.

* Do we want to call it the competitor booklet as it also has info for non competitors e.g. team leaders and people looking after the kids (forget the right word for this) would event booklet be better (or do we even need to say what it is on the cover).
* I believe there are plans to add a persistent link to the rules that we can use so the game page doesn't go out of date for those who keep the booklet as a souvenir. @PeterJCLaw do we have a confirmed url for this yet
* Per comments on slack i think the food map will also want the details for people doing deliver to the venue.
* I think we can do something more with the cover than just the plain blue background but I am aware we are short of time to get this done.

Generally looking really good thanks for compiling this.

**Comment by PeterJCLaw** [comment]
Overall looks good, thanks!

* We advertise our social media, but not our own website -- please could we encourage people to look at the website? The following pages will be particularly useful to the teams:
  * [srobo.org/comp/team](http://srobo.org/comp/team) (has information about their matches plus summary of their position & points)
  * [srobo.org/comp/schedule](http://srobo.org/comp/schedule) (has the match schedule)
  * The other /comp/ pages they might want are [srobo.org/comp/league](http://srobo.org/comp/league) (league points overview) and [srobo.org/comp/knockout](http://srobo.org/comp/knockout) (knockout structure), but the first two are the key ones; all of these pages are live-updating during the competition
  * The homepage will have the livestream plus an overview of the matches, though that's more aimed at guests and external viewers
* (as Dan noted the food map has gone awry for me too)
* Who is this Jake Howard person anyway?

Also some nits:
* on page 3, maybe drop the underline under the first line "The following rules..."? This is definitely personal preference, so feel free to ignore.
* on page 3, could we have the list entries wrap to the same indent as their first lines? (Probably just preference, but I find that that's much neater)
* on page 5, "and havent competed before" should have an apostrophe: "and haven't competed before"

Regarding the WiFi credentials, I had previously assumed that we'd hand those out at reception. Perhaps in exchange for collecting the battery/charger/etc. from the team. Is there a reason not to do that?

I had been assuming that the plan was to have a single copy per team, but I've just realised that that was an assumption and I don't know what others had in mind?

> * I believe there are plans to add a persistent link to the rules that we can use so the game page doesn't go out of date for those who keep the booklet as a souvenir. @PeterJCLaw do we have a confirmed url for this yet (see [srobo/website#29](https://github.com/srobo/website/issues/29))

We could add the full link to the rules, but I'm not sure that would fit. I think keeping it short is better. I think it would be easy to add something to the page along the lines of "Looking for older rulebooks? Go here...".

**Comment by RealOrangeOne** [comment]
> Regarding the WiFi credentials

There'll be 1 set of credentials per team. I think the current plan is to exchange them for batteries and chanrgers, which will be done at reception, or somewhere nearby (also possibly helpdesk depending on space).

**Comment by PeterJCLaw** [comment]
I like the idea of it being an exchange :+1: 
I'd encourage doing it as close to reception as possible -- so as to reduce the chances that someone sneaks past even unintentionally. I think we've historically done this as part of reception, but I'm not sure as I've never been on reception.

**Comment by Tyler-Ward** [comment]
I believe it was suggested at the weekend to do the swap and credentials issue at help-desk as they will be handling the batteries anyway.

**Comment by TomWheal** [comment]
[Booklet-v02.pdf](https://github.com/srobo/tasks/files/3017133/Booklet-v02.pdf)

**Comment by antoinepetty** [comment]
A few minor adjustments but looks good:

- I don't think the "competitor booklet" text on the front is centered.
- Similarly, the quotes page seems to be shifted to the right. 
- On the quotes page we should be using competition notation as discussed in Slack. So instead of "CLY 14-15 Blue Shirt 16-19" it should be "CLY SR2015 Blue Shirt SR2017-19"

**Comment by richardbarlow** [comment]
On page 9 s/your/you're/ in the league link description

**Comment by TomWheal** [comment]
All the Comments Above have now been fixed along with some others I noticed
[Booklet-v03.pdf](https://github.com/srobo/tasks/files/3017831/Booklet-v03.pdf)

**Comment by Tyler-Ward** [comment]
Looks good, 
A quick to do idea for the cover that will break up the blue while still keeping it clean would be to copy the background design used on the business cards and name badges with the whited out logo in the background.

**Comment by TomWheal** [comment]
[Booklet-Final.pdf](https://github.com/srobo/tasks/files/3018129/Booklet-Final.pdf)
Printing will start tomorrow, can change design until then.

**Comment by antoinepetty** [comment]
Final nitpick for the quotes page if not too late: 
The SR year notation is a single year in which the competition is held. So competing 2015-16 means putting SR2016

I can live with it's current form though ahah. Excellent work overall!

**Comment by TomWheal** [comment]
Its 2 years is why I put it like that, so really they were competing for 2014-16, There's not really space for SR2015, SR2016

**Comment by kierdavis** [comment]
There's a few typos on the "Useful Links" page:
* "Useful for seeing who your’e up against!": *your'e -> you're*
* "All these pages will be automatically updated throughout the compeition!": *compeition -> competition*

Otherwise it looks good to me.

**Comment by PeterJCLaw** [comment]
@TomWheal thanks for doing this. Please could you contribute the source into a suitable repo? (Probably https://github.com/srobo/promotional-material?)

I suspect I'm too late for changes, but notes for the future:

I agree that the "SRYYYY-YY" notation is confusing. Especially as we already imbue the idea of two calendar years into "SRYYYY", it's entirely unclear what it means.

There wasn't an SR2007; SR2008 was the first. I think it's unclear to use competition years to denote volunteer year ranges -- volunteers come and go throughout (and many have volunteered during periods when there weren't competitions), so using calendar years is clearer.

Expressing that someone is _still_ in a role is best done with "YYYY - present" rather than putting an end year. An explicit end year implies that they're no longer in that role.

"Blueshirt" has tended to be rendered as a single word.

If we're going to get quotes from people, please don't alter them without consultation. [Mine](https://studentrobotics.slack.com/archives/C0EEJ0ASY/p1553719413018200) has been edited and has lost the key part -- that the things volunteers work on are used by a large audience and have to be reliable. That's the part which is impressive to employers, the deadline part is secondary.

**Comment by TomWheal** [comment]
@PeterJCLaw  the source can be found at, https://github.com/srobo/competitor-booklet-2019/releases , Ill work on converting these too svgs however as it was designed in inDesign It might take a little while. 

On your other comments, ill add them to an issue in that repo so they don't get lost for future years. 
In the future a longer proofing time is probably needed, that would have allowed us to catch those issues.

**Comment by PeterJCLaw** [comment]
Ah, I see.

I think it's important that we have a small number of repos which gather together similar things. I'd definitely encourage that we _not_ have year-specific repos unless they're really necessary.

Keeping things in a small number of repos will make it easy for people to find similar things in the future as well as know where to put things. I appreciate that this may not have been something which was particularly well communicated before you started though.

**Comment by TomWheal** [comment]
@PeterJCLaw 
Probably best to refer this too @Tyler-Ward as he created the repo, not me.
